packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -0,0 +1,416 @@
+
+The example below uses `onAction` to tell the user which Menu item was selected.
+
+```tsx example
+function Example() {
+  let onAction = (value) => alert('Menu item selected: ' + value);
+
+  return (
+    <div>
+      <MenuTrigger>
+        <ActionButton>
+            Menu Button
+        </ActionButton>
+        <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name" onAction={onAction}>
+          {item => <Item>{item.name}</Item>}
+        </Menu>
+      </MenuTrigger>
+    </div>
+  );
+}
+```
+
+## Props
+
+### MenuTrigger
+
+<PropTable component={docs.exports.MenuTrigger} links={docs.links} />
+
+### Menu
+
+<PropTable component={docs.exports.Menu} links={docs.links} />
+
+## Visual Options
+
+### Align and Direction
+
+[View guidelines](https://spectrum.adobe.com/page/popovers/#Placement)
+
+```tsx example
+<MenuTrigger align="start" direction="bottom">
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger align="start" direction="top">
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger align="end" direction="bottom">
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger align="end" direction="top">
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+```
+
+### Autofocus
+
+Applying `autoFocus` to the Menu of the MenuTrigger sets focus to the Menu upon opening.
+
+```tsx example
+<MenuTrigger closeOnSelect={false}>
+  <ActionButton>
+      Menu Button Autofocus
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    autoFocus>
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger closeOnSelect={false}>
+  <ActionButton>
+      Menu Button Autofocus Last
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    autoFocus="last">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+```
+
+### Closes Menu onSelect
+
+```tsx example
+<MenuTrigger closeOnSelect={false}>
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
```

Comment:
I think it would be good to add a `selectionMode` to this Menu example as well since it makes it more apparent that a selection is happening and that the menu is still staying open despite of the selection changes